glance/manifests/api.pp

```diff
@@ -239,6 +239,21 @@
     warning('The mysql_module parameter is deprecated. The latest 2.x mysql module will be used.')
   }
 
+  if ($auth_protocol == 'https') {
+    class {'moc_openstack::ssl::add_glance_cert':
+      require => File['/etc/glance'],
+    }
+    class {'moc_openstack::ssl::temp_glance_fix':
```

Comment:
It fails for compute since /etc/glance is not present on compute. Need to find a better fix for this.